web/app/Repository/GameRepository.php

```diff
@@ -71,6 +59,12 @@ public function storeGame(array $post): bool
         $id_white = $post["player_white"];
         $id_black = $post["player_black"];
 
+        echo "<h1>"."Spiel wurde eingetragen"."</h1>";
```

Comment:
prefer single quotes ' everywhere. double quotes will be evaluated (variables in it will be parsed) and can easily result in unwanted behaviour.

Reply:
I will look after it in the future